app/helpers/messages_helper.rb

```diff
@@ -1,2 +1,18 @@
 module MessagesHelper
+    # Extract the visible email content from HTML, removing meta tags, style tags, etc.
```

Comment:
@godie like you have this helper for rendering the email inside the Violet Rails system, you'll need a separate rendering strategy in app/views/e_mailer/ship.html.erb to render the revolvapp HTML without trix tags

Reply:
well the helper function is doing it, i added the helper to the mailer to reuse, can you validate ship.html.erb is sending the rigth format